examples/charm++/jacobi2d-2d-decomposition/jacobi2d.ci

```diff
@@ -17,7 +17,7 @@ mainmodule jacobi2d {
   array [2D] Jacobi {
     entry Jacobi(void);
     entry void begin_iteration(void);
-    entry void receiveGhosts(int iter, int dir, int size, double ghosts[size]);
+    entry void receiveGhosts(int iter, int dir, int size, std::vector<double> ghosts);
```

Comment:
Sending an `std::vector` via entry method call will result in generally worse performance; it results in an unnecessary/unavoidable copy. It would be better to preserve `receiveGhosts` as it is, and simply send the `.data()` of the `std::vector` in user-code.